data/.schema.yaml

```diff
@@ -129,13 +129,37 @@ properties:
               enum:
                 - female
                 - male
+                - unknown
             age:
               description: Age of the participant in years.
               oneOf:
                 - type: number
                   minimum: 0
                   maximum: 120
                 - const: unknown
+            race:
+              description: Race of the participant.
+              type: string
+              enum:
+                - asian
+                - black
+                - white
+                - other
+                - unknown
+            ethnicity:
+              description: Ethnicity of the participant.
+              type: string
+              enum:
+                - hispanic
+                - not hispanic
+                - unknown
```

Comment:
Ethnicity and race are such tricky concepts. The hispanic/not hispanic dichotomy is quite common in the United States, but ethnicity is used differently elsewhere in the world because culture and race are often so strongly connected. How do you think we should encode multi-racial participants?

Reply:
This is a great question. Currently,  I do not have a good answer to it. We should have a better understanding about how they were coded in different studies. After that, maybe we can consult some experts at Emory or Harvard about this.

Reply:
Another option for encoding the US-based dichotomy could be `hispanic: {true, false, unknown}`?

Reply:
As we just discussed, let's just keep it as `hispanic`, `not hispanic`, and `unknown` for now.